src/aap_eda/services/ruleset/activation_kubernetes.py

```diff
@@ -415,19 +415,21 @@ def set_activation_status(
         self, instance: models.ActivationInstance, status: ActivationStatus
     ) -> None:
         try:
+            now = timezone.now()
             instance.status = status
-            instance.updated_at = timezone.now()
+            instance.updated_at = now
```

Comment:
I wonder why we don't set this field with `auto_now=True` is there a reason for it?

Reply:
The value can be set by an external value https://github.com/ansible/eda-server/blob/main/src/aap_eda/wsapi/consumers.py#L151